### PR TITLE
Added <kbd> notations for keyboard keys.

### PR DESCRIPTION
**Pull request by bantya**
## PR Checklist
* [ ] Closes #xxx
* [ ] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [ ] Tests added/passed
* [x] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

### CODE REVIEW

**Comment by DHowett-MSFT** [comment]
How do you feel about <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>X</kbd> versus <kbd>Ctrl+Shift+X</kbd>? I somewhat like the latter, as it seems more concise, but it's definitely not "a single keycap".

File: doc/user-docs/index.md [hunk 1]
```diff
@@ -31,8 +31,8 @@ None at this time. See issue [#607](https://github.com/microsoft/terminal/issues
 
 ## Multiple Tabs
 
-Additional shells can be started by hitting the `+` button from the tab bar -- a new instance of the
-default shell is displayed (default shortcut `Ctrl+Shift+1`).
+Additional shells can be started by hitting the <kbd>+</kbd> button from the tab bar -- a new instance of the
```
[the remark this answers is not in the document]
Comment (bantya) [reply doc/user-docs/index.md]: Oops, this is why you should not code late at night! 🥱
I accept, I got this wrong. 😞
Answer to your question is: **no**, `+` is perfect for this case.

**Comment by bantya** [comment]
> 
> 
> How do you feel about <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>X</kbd> versus <kbd>Ctrl+Shift+X</kbd>? I somewhat like the latter, as it seems more concise, but it's definitely not "a single keycap".

In <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>X</kbd>'s case one can clearly understand the keys in a single glance, whereas for <kbd>Ctrl+Shift+X</kbd>, the `+` in between the keys makes it slightly harder to read and requires a bit more time to parse the keystrokes.

So, according to me:
<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>X</kbd>: 👍 
<kbd>Ctrl+Shift+X</kbd>: 👎

File: doc/user-docs/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Luuk34) [reply doc/user-docs/index.md]: > default shell is displayed (default shortcut `Ctrl+Shift+1`).
The default of a default can be changed.
My `defaultProfile` does point to a profile with the shortcut `Ctrl+Shift+2`

**Comment by bantya** [comment]
@zadjii-msft
Changed the `<kbd>` to `<code>`. It's OK now!
👍

**Comment by miniksa** [comment]
> @DHowett-MSFT Dustin Howett FTE
> 
> > How do you feel about Ctrl+Shift+X versus Ctrl+Shift+X? I somewhat like the latter, as it seems more concise, but it's definitely not "a single keycap".
> 
> In Ctrl+Shift+X's case one can clearly understand the keys in a single glance, whereas for Ctrl+Shift+X, the `+` in between the keys makes it slightly harder to read and requires a bit more time to parse the keystrokes.
> 
> So, according to me:
> Ctrl+Shift+X: 👍
> Ctrl+Shift+X: 👎

I agree with this philosophy. I'd prefer each key is its own.

**Comment by miniksa** [comment]
Admin override because it's a doc change and I don't want to sit for CI. Thanks for your contribution.

**Pull request merged**